Makefile

```diff
@@ -1,5 +1,5 @@
 BUILD_DIR ?= /build
-TEMPLATE_EXCLUDE += c2cgeoportal/scaffolds tests/testegg c2cgeoportal/templates ngeo
+TEMPLATE_EXCLUDE += .build .venv c2cgeoportal/scaffolds tests/testegg c2cgeoportal/templates ngeo
```

Comment:
=> TEMPLATE_EXCLUDE += c2cgeoportal/scaffolds tests/testegg c2cgeoportal/templates

Reply:
It was you who added it. For avoiding trouble when we have a .venv for the IDE.
Removing .build, though

Reply:
see FIND_OPTS comment to solve that :-)

Reply:
With this modif, the tests are failing...

Reply:
and with 
TEMPLATE_EXCLUDE += scaffolds templates
FIND_OPTS = tests/functional doc c2cgeoportal .tx $(...
?